examples/dgmulti_1d/elixir_euler_shu_osher_gauss_shock_capturing.jl

```diff
@@ -0,0 +1,97 @@
+using Trixi
+using OrdinaryDiffEq
+
+γ_gas = 1.4
```

Comment:
```suggestion
gamma_gas = 1.4
```

Reply:
We generally try to avoid unicode for accessibility purposes

Reply:
Got it, modified